Server/db/MongoDB.js

```diff
@@ -284,6 +303,21 @@ const deleteAllMonitors = async (req, res) => {
   }
 };
 
+/**
+ * Delete all monitors associated with a user ID
+ * @async
+ * @param {string} userId - The ID of the user whose monitors are to be deleted.
+ * @returns {Promise} A promise that resolves when the operation is complete.
+ */
+const deleteMonitorsByUserId = async (userId) => {
+  try {
+    const result = await Monitor.deleteMany({ userId: userId });
```

Comment:
When a monitor is deleted it also must be removed from the `JobQueue` or the job will continue to run.

Please see the `deleteMonitor` method in `monitorController` to see how this works.